pkg/kubelet/kubelet.go

```diff
@@ -368,21 +364,19 @@ func NewMainKubelet(
 	}
 	klet.containerManager = containerManager
 
-	go util.Until(klet.syncNetworkStatus, 30*time.Second, util.NeverStop)
-	if klet.kubeClient != nil {
-		// Start syncing node status immediately, this may set up things the runtime needs to run.
```

Comment:
It'd be greatly appreciated If you can add some comments about the startup sequence, so that the next person who touches the code would be aware of the potential issues.